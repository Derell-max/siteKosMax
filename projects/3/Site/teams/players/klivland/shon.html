<html>
 <head>
 <meta charset="Windows-1251">
 <title>Шон Мэрион</title>
 <style>
  body {
   background-image: url(../../../images/background.jpg);
   background-attachment: fixed;
   background-size: cover;
  }
 </style>
 </head>
 <body>
  <div align="center">
   <a href="../../../index.html"><img src="../../../images/nba.png"></a>
   <br></br>
  </div>
  <a href="../../../teams/klivland.html"><table bgcolor="white"><tr><td><font face="impact" color="red">&#9668;Назад</font></td></tr></table></a>
  <br></br>
  <div align="left">
   <table bgcolor="black">
    <tr>
     <td rowspan="9"><font face="impact" color="white"><img src="../../../teams/images/klivland/15.jpg"></td>
     <td><font face="impact" color="white">Матчи</font></td>
     <td><font face="impact" color="white">57</font></td>
     <td><font face="impact" color="white"></font></td>
     <td><font face="impact" color="white">Передачи (всего/среднее)</font></td>
     <td><font face="impact" color="white">51</font></td>
     <td><font face="impact" color="white">0.9</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">В основе</font></td>
     <td><font face="impact" color="white">24</font></td> 
     <td><font face="impact" color="white"></font></td>
     <td><font face="impact" color="white">Подборы в защите (всего/среднее)</font></td>
     <td><font face="impact" color="white">138</font></td>   
     <td><font face="impact" color="white">2.4</font></td>  
    </tr>
    <tr>
     <td><font face="impact" color="white">Время (всего/среднее)</font></td>
     <td><font face="impact" color="white">1100:49</font></td>
     <td><font face="impact" color="white">19:19</font></td>
     <td><font face="impact" color="white">Подборы в атаке (всего/среднее)</font></td>
     <td><font face="impact" color="white">64</font></td>
     <td><font face="impact" color="white">1.1</font></td>   
    </tr>
    <tr>
     <td><font face="impact" color="white">Очки (всего/среднее)</font></td>
     <td><font face="impact" color="white">276</font></td>
     <td><font face="impact" color="white">4.8</font></td> 
     <td><font face="impact" color="white">Подборы (всего/среднее)</font></td>
     <td><font face="impact" color="white">202</font></td> 
     <td><font face="impact" color="white">3.5</font></td>    
    </tr>
    <tr>
     <td><font face="impact" color="white">2-очковые броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">107/221</font></td>
     <td><font face="impact" color="white">1.9/3.9</font></td> 
     <td><font face="impact" color="white">Перехваты (всего/среднее)</font></td>
     <td><font face="impact" color="white">27</font></td> 
     <td><font face="impact" color="white">0.5</font></td>    
    </tr>
    <tr>
     <td><font face="impact" color="white">2-очковые броски (% реализации)</font></td>
     <td><font face="impact" color="white">48.4%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Потери (всего/среднее)</font></td>
     <td><font face="impact" color="white">33</font></td>  
     <td><font face="impact" color="white">0.6</font></td>   
    </tr>
    <tr>
     <td><font face="impact" color="white">3-очковые броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">12/46</font></td>
     <td><font face="impact" color="white">0.2/0.8</font></td> 
     <td><font face="impact" color="white">Блокшоты (всего/среднее)</font></td>
     <td><font face="impact" color="white">27</font></td>
     <td><font face="impact" color="white">0.5</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">3-очковые броски (% реализации)</font></td>
     <td><font face="impact" color="white">26.1%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Блокшоты соперника (всего/среднее)</font></td>
     <td><font face="impact" color="white">22</font></td>
     <td><font face="impact" color="white">0.4</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">Штрафные броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">26/34</font></td>
     <td><font face="impact" color="white">0.5/0.6</font></td> 
     <td><font face="impact" color="white">Фолы (всего/среднее)</font></td>
     <td><font face="impact" color="white">58</font></td>   
     <td><font face="impact" color="white">1</font></td>  
    </tr>
    <tr>
     <td align="center"><font face="impact" color="white">Шон Мэрион</font></td>
     <td><font face="impact" color="white">Штрафные броски (% реализации)</font></td>
     <td><font face="impact" color="white">76.5%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Коэффициент полезности (всего/среднее)</font></td>
     <td><font face="impact" color="white">314</font></td>  
     <td><font face="impact" color="white">5.5</font></td>   
    </tr>
   </table>
  </div> 
 </body> 
</html>