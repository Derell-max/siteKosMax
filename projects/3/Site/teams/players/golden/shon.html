<html>
 <head>
 <meta charset="Windows-1251">
 <title>Шон Ливингстон</title>
 <style>
  body {
   background-image: url(../../../images/background.jpg);
   background-attachment: fixed;
   background-size: cover;
  }
 </style>
 </head>
 <body>
  <div align="center">
   <a href="../../../index.html"><img src="../../../images/nba.png"></a>
   <br></br>
  </div>
  <a href="../../../teams/golden.html"><table bgcolor="white"><tr><td><font face="impact" color="red">&#9668;Назад</font></td></tr></table></a>
  <br></br>
  <div align="left">
   <table bgcolor="black">
    <tr>
     <td rowspan="9"><font face="impact" color="white"><img src="../../../teams/images/golden/4.jpg"></td>
     <td><font face="impact" color="white">Матчи</font></td>
     <td><font face="impact" color="white">78</font></td>
     <td><font face="impact" color="white"></font></td>
     <td><font face="impact" color="white">Передачи (всего/среднее)</font></td>
     <td><font face="impact" color="white">256</font></td>
     <td><font face="impact" color="white">3.3</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">В основе</font></td>
     <td><font face="impact" color="white">2</font></td> 
     <td><font face="impact" color="white"></font></td>
     <td><font face="impact" color="white">Подборы в защите (всего/среднее)</font></td>
     <td><font face="impact" color="white">139</font></td>   
     <td><font face="impact" color="white">1.8</font></td>  
    </tr>
    <tr>
     <td><font face="impact" color="white">Время (всего/среднее)</font></td>
     <td><font face="impact" color="white">1468:15</font></td>
     <td><font face="impact" color="white">18:49</font></td>
     <td><font face="impact" color="white">Подборы в атаке (всего/среднее)</font></td>
     <td><font face="impact" color="white">43</font></td>
     <td><font face="impact" color="white">0.6</font></td>   
    </tr>
    <tr>
     <td><font face="impact" color="white">Очки (всего/среднее)</font></td>
     <td><font face="impact" color="white">461</font></td>
     <td><font face="impact" color="white">5.9</font></td> 
     <td><font face="impact" color="white">Подборы (всего/среднее)</font></td>
     <td><font face="impact" color="white">182</font></td> 
     <td><font face="impact" color="white">5.3</font></td>    
    </tr>
    <tr>
     <td><font face="impact" color="white">2-очковые броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">198/394</font></td>
     <td><font face="impact" color="white">2.5/5.1</font></td> 
     <td><font face="impact" color="white">Перехваты (всего/среднее)</font></td>
     <td><font face="impact" color="white">50</font></td> 
     <td><font face="impact" color="white">0.6</font></td>    
    </tr>
    <tr>
     <td><font face="impact" color="white">2-очковые броски (% реализации)</font></td>
     <td><font face="impact" color="white">50.3%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Потери (всего/среднее)</font></td>
     <td><font face="impact" color="white">102</font></td>  
     <td><font face="impact" color="white">1.3</font></td>   
    </tr>
    <tr>
     <td><font face="impact" color="white">3-очковые броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">0/2</font></td>
     <td><font face="impact" color="white">0/0</font></td> 
     <td><font face="impact" color="white">Блокшоты (всего/среднее)</font></td>
     <td><font face="impact" color="white">20</font></td>
     <td><font face="impact" color="white">0.3</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">3-очковые броски (% реализации)</font></td>
     <td><font face="impact" color="white">0%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Блокшоты соперника (всего/среднее)</font></td>
     <td><font face="impact" color="white">16</font></td>
     <td><font face="impact" color="white">0.2</font></td>     
    </tr>
    <tr>
     <td><font face="impact" color="white">Штрафные броски (всего/среднее)</font></td>
     <td><font face="impact" color="white">65/91</font></td>
     <td><font face="impact" color="white">0.8/1.2</font></td> 
     <td><font face="impact" color="white">Фолы (всего/среднее)</font></td>
     <td><font face="impact" color="white">110</font></td>   
     <td><font face="impact" color="white">1.4</font></td>  
    </tr>
    <tr>
     <td align="center"><font face="impact" color="white">Шон Ливингстон</font></td>
     <td><font face="impact" color="white">Штрафные броски (% реализации)</font></td>
     <td><font face="impact" color="white">71.4%</font></td>
     <td><font face="impact" color="white"></font></td> 
     <td><font face="impact" color="white">Коэффициент полезности (всего/среднее)</font></td>
     <td><font face="impact" color="white">517</font></td>  
     <td><font face="impact" color="white">6.6</font></td>   
    </tr>
   </table>
  </div> 
 </body> 
</html>